packages/compass/src/main/application.ts

```diff
@@ -97,7 +99,7 @@ class CompassApplication {
     this.setupLifecycleListeners();
     this.setupApplicationMenu();
     this.setupWindowManager();
-    this.trackApplicationLaunched(globalPreferences);
+    void this.trackApplicationLaunched(globalPreferences);
```

Comment:
This goes from having a a function that can never throw to one that potentially can throw and the rejection would be unhandled, are we sure?

Reply:
More of a nit, feel free to keep it this way or change it and merge: now this is not going to cause problems anymore, but if it was already in this way I would have never spotted this same issue. 

I wonder if it would be better to just `.catch()` explicitly here instead.

Reply:
mm actually yeah on a second though .. i don't know .. i think that we should not have this `void` here. We may add a line of code throwing inside `trackApplicationLaunched` and never notice.

Reply:
[via chat] Maurizio helped me realize that `.track` function takes a callback. I used that to determine the track event props.